### PR TITLE
added a recommended read

### PR DESCRIPTION
**Pull request by SteveLee1287**
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ Recommended Training and Reading
 - [PowerShell Quick Reference Guides][quick-reference] by PowerShellMagazine.com
 - [PowerShell 5 How-To Videos][script-guy-how-to] by Ed Wilson
 - [PowerShell Documentation](https://docs.microsoft.com/powershell)
+- [Learn Windows PowerShell in a Month of Lunches][month-of-lunches]
```
Comment (iSazonov) [inline docs/learning-powershell/README.md]: Should it be in Commercial Resources?

Comment (SteveLee1287) [reply docs/learning-powershell/README.md]: I agree, can I move it without performing a new pull request?

File: docs/learning-powershell/README.md [hunk 2]
```diff
@@ -135,3 +136,4 @@ Commercial Resources
 [quick-reference]: http://www.powershellmagazine.com/2014/04/24/windows-powershell-4-0-and-other-quick-reference-guides/
 [script-guy-how-to]:https://blogs.technet.microsoft.com/tommypatterson/2015/09/04/ed-wilsons-powershell5-videos-now-on-channel9-2/
 [basic-cookbooks]:https://docs.microsoft.com/en-us/powershell/scripting/getting-started/fundamental/sample-scripts-for-administration?view=powershell-6
+[month-of-lunches]:https://www.amazon.com/Learn-Windows-PowerShell-Month-Lunches/dp/1617291080
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/README.md]: Looks like there is a newer 3rd edition: https://www.amazon.com/Learn-Windows-PowerShell-Month-Lunches/dp/1617294160

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by TravisEz13** [comment]
There is a duplicate PR:  https://github.com/PowerShell/PowerShell/pull/8067

**Comment by adityapatwardhan** [comment]
Closing this PR as the README now has a link to the 3rd edition of the book.

**Pull request closed without merging**